### PR TITLE
Issue 357 - Addressed API and deployment readme review comments (For Release branch)

### PR DESCRIPTION
**Pull request by chinmayic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,36 +3,130 @@
...
+
+
+##  Resource Aggregator for ODIM logical architecture
+
+Resource Aggregator for ODIM framework adopts a layered architecture and has many functional layers.
+
+
+The following figure shows these functional layers of Resource Aggregator for ODIM deployed in a data center.
+
+![ODIM_architecture](images/arch.png)
```
Comment (ashraf-vazeer) [inline README.md:62]: This link is now working.

File: README.md [hunk 2]
```diff
@@ -3,36 +3,130 @@
...
+
+- **Services layer**
+
+
+The services layer is where all the services are hosted. This layer implements service logic for all use cases
+through an extensible domain model (Redfish Data Model). Requests coming from the API layer and the
+responses coming from the plugin layer are mapped to the actual end resources in this layer. It maintains the state
+for event subscriptions, credentials, and tasks. It also hosts a message bus called the Plug-in Message Bus (PMB).
+
+![Redfish_data_model](images/redfish_data_model.png)
```
Comment (ashraf-vazeer) [inline README.md:81]: This link is now working.

**Review by ashraf-vazeer — CHANGES_REQUESTED** [review]
please address the comments

File: README.md (hunk 1 above)
Comment (chinmayic) [reply README.md:62]: It will work once it is merged to this branch.

File: README.md (hunk 2 above)
Comment (chinmayic) [reply README.md:81]: It will work once it is merged to this branch.

**Review by ashraf-vazeer — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**